src/test/java/seedu/linkedout/model/applicant/JobTest.java

```diff
@@ -30,7 +92,27 @@ public void isValidJob() {
 
```

Comment:
What i meant above, this also has the whitespace test
```suggestion
        assertFalse(Job.isValidJob("#")); // non-alphanumeric characters
        assertFalse(Job.isValidJob(":")); 
```